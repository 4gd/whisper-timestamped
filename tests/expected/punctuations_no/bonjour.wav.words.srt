1
00:00:00,220 --> 00:00:00,440
Bonjour

