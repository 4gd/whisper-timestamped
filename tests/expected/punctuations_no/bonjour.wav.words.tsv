start	end	text
160	780	Bonjour
